python/python/bystro/ancestry/preprocess_1kgp_using_gnomad_loadings.sh

```diff
@@ -0,0 +1,37 @@
+#Extract gnomad loadings variant list from 1kgp genomes using plink
+
+#TODO convert this process to one using our own vcf parser
+
+#Download plink2 - can be downloaded with example below or at https://www.cog-genomics.org/plink/2.0/
+#wget https://s3.amazonaws.com/plink2-assets/plink2_linux_avx2_20230707.zip
+#unzip plink2_linux_avx2_20230707.zip 
+
+#Download 1kgp genomes - make sure this is the most recent version
+wget 'ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20220422_3202_phased_SNV_INDEL_SV/1kGP_high_coverage_Illumina.chr'*
+
+#Gnomad loadings have been preprocessed to extract the variant list only as gnomadvariantlist.txt
+
+#Extract from each autosomal chromosome for ancestry
+for ((chr=1; chr<=22; chr++))
+do
+    #Input and output paths for each chromosome
+    input_vcf="1kGP_high_coverage_Illumina.chr${chr}.filtered.SNV_INDEL_SV_phased_panel.vcf.gz"
+    output_base="chr${chr}"
+
+    #Run plink2 with current chromosome
+    ./plink2 --extract gnomadvariantlist.txt \
+             --make-pgen \
+             --out "$output_base" \
+             --vcf "$input_vcf"
+     # Append output file name to the merge list, excluding 'chr1'
```

Comment:
this seems a little more error-prone than writing each file to a tmp dir and concatenating them all at the end with `vcfcat`-- what happens to the header lines for chrs2-22?

Reply:
The vcf commented headers aren't all kept through the plink processing as it's converted to plink format to do so, but I'm not sure exactly what you mean here. Are you suggesting not using the plink process at all here? I just wanted to document this method for this preprocess step that is used for this version of 1kgp until this step is converted to our own vcf parser module.

Reply:
ok, if there are no header lines I withdraw my comment :)